libraries/Microsoft.Bot.Builder.Dialogs.Debugging/DebugAdapter.cs

```diff
@@ -479,7 +480,10 @@ private async Task<Protocol.Message> DispatchAsync(Protocol.Message message, Can
             }
             else if (message is Protocol.Request<Protocol.Disconnect> terminate)
             {
-                this.terminate();
+                if (this.terminate != null)
+                {
+                    this.terminate();
+                }
```

Comment:
If process does not exit, files will remain locked and future launches will fail.